code/__DEFINES/dcs/signals.dm

```diff
@@ -167,6 +167,11 @@
 	#define COMPONENT_NO_ATTACK_HAND (1<<0)								//works on all 3.
 //This signal return value bitflags can be found in __DEFINES/misc.dm
 
+///from base of atom/expose_reagents():
+#define COMSIG_ATOM_EXPOSE_REAGENTS "atom_expose_reagents"
+	/// Prevents the atom from being exposed to reagents if returned on [COMPONENT_ATOM_EXPOSE_REAGENTS]
+	#define COMPONENT_NO_EXPOSE_REAGENTS (1<<1) // Different from [TRUE] for QOL
```

Comment:
```suggestion
	#define COMPONENT_NO_EXPOSE_REAGENTS (1<<0) // Different from [TRUE] for QOL
```
Start with `1`

I don't see this ever being used, nor anything registering to `COMSIG_ATOM_EXPOSE_REAGENTS`, am I missing something?

Reply:
It was supposed to be a loose hook for someone else to take advantage of if they wanted to make teflon coating or something else that makes people immune to getting exposed or does something when you get exposed. I can remove it if you want me to.